internal/commands/user/output.go

```diff
@@ -16,6 +16,18 @@ const (
 	headerDetails                = "Details"
 )
 
+type userOutputs []userOutput
+
+func (uo userOutputs) outputsByProviderType() map[realm.AuthProviderType]userOutputs {
+	var outputsByProviderType = map[realm.AuthProviderType]userOutputs{}
+	for _, output := range uo {
```

Comment:
can we make this read `for _, output := range outputs {`?  and change `outputsByProviderType` to just `out` or `outputsM` or something?